src/.vuepress/layouts/Landing.vue

```diff
@@ -34,31 +34,31 @@
 		<!--<img src="/top-corner-2.svg" class="corner2">-->
 		<MkDots v-parallax="1.4" class="dots1" :space="30"/>
 		<MkDots v-parallax="1.5" class="dots2" :space="30"/>
-		<img v-parallax="2" src="/screenshot-desktop.png" class="screenshot desktop">
-		<img v-parallax="3" src="/screenshot-mobile.png" class="screenshot mobile">
-		<img v-parallax="4" src="/ai.png" class="ai">
+		<img v-parallax="2" src="/screenshot-desktop.png" class="screenshot desktop" alt="Desktop screenshot">
+		<img v-parallax="3" src="/screenshot-mobile.png" class="screenshot mobile" alt="Mobile screenshot">
+		<img v-parallax="4" src="/ai.png" class="ai" alt="Ai-chan!">
```

Comment:
Suggestions about Improving your sentences:

Desktop screenshot → Screenshot of Misskey on desktop
Mobile screenshot → Screenshot of Misskey on mobile
Ai-chan! → Ai-chan Misskey Official Mascot character